src/library/searchquery.cpp

```diff
@@ -7,6 +7,12 @@
 #include "library/dao/trackschema.h"
 #include "util/db/sqllikewildcards.h"
 
+namespace {
+
+const bool sDebug = false;
```

Comment:
I don't like this kind of hard-coded switches in the code. Unfortunately I noticed that our Logging class does not offer a function to query the current log level! I will shortly publish a small PR that adds the missing functionality.